### ISSUE TITLE
Incorrect threshold value for tmax to intensity ratio for sigmoidal fit

### ISSUE THREAD

**Issue by empetroni** [wilkelab/sicegar#3]
In the "categorize.R" function, I noticed that on line 157, when the function checks to see if the tmax to intensity ratio for a sigmoid curve is above the threshold value, the threshold value for double sigmoidal is used instead of the threshold for sigmoidal.

**Comment by clauswilke** [comment]
Yes, looks that way. Feel free to submit a pull request to fix this. If you do, please also adjust the default value for `threshold_sm_tmax_IntensityRatio` so behavior of the package doesn't actually change.
